airgun/entities/subscription.py

```diff
@@ -39,6 +39,8 @@ def _wait_for_process_to_finish(self, name, has_manifest=False):
                 handle_exception=True, timeout=10,
                 logger=view.logger
         )
+        # uncomment when BZ#1651981 is fixed
```

Comment:
what is a reason for that change? Does someone approve that workaround?
If all cases for subscription fail, so bug is a blocker and only after confirmation you can do changes like this

Reply:
I'm not sure it's a good idea to fail all the tests which are using subscriptions page when all the functionality being tested is working ok.

Reply:
it is not okay, as user will see 404 error on the screen. We cannot omit that message. For WebUI that  issue is a blocker, so we cannot test functionality if we see anything like this. Except the case when everybody in team is informed and team lead is approve to skip that error